### PR TITLE
Added readme, minor fixes.

### PR DESCRIPTION
**Pull request by muskanbagrecha**

### CODE REVIEW

**Comment by shruthipatil11** [comment]
You can have cdn link of font awesome in common css file instead of using script in each html pages going forward.
Over all code is clean and correct. Good to merge

**Review by shruthipatil11 — APPROVED** [review]

**Review by ritikaawasthy — APPROVED** [review]
looks good to merge

File: js/filters.js [hunk 1]
```diff
@@ -18,12 +18,15 @@ const collapseFilter = () => {
...
-    const categoryFilter = document.querySelector(".category-filter");
-    const priceRange = document.querySelector(".price-range");
+const clearFilters = () => {
+    const clearFilterBtn = document.querySelector(".filter__clear");
+    clearFilterBtn.addEventListener("click", () => {
+        const data = document.querySelectorAll(".filter-data");
+        data[0].reset();
+        data[1].reset();
+    })
 }
```
Comment (PuneetP16) [inline js/filters.js:28]: Suggestion: Add a comment on what this function is doing maybe.

File: readme.md [hunk 2]
```diff
@@ -0,0 +1,101 @@
+# Fashgram Store - where creative meets Fashion
+
```
Comment (PuneetP16) [inline readme.md:2]: Readme looks well documented, Suggestion, you could add a video maybe showing your quiz app.

**Review by PuneetP16 — APPROVED** [review]
Your quiz app theme is too consistent with your theme. Nice work! Good to merge!

**Pull request merged**